packages/components/datepicker/src/components/osds-datepicker/core/controller.ts

```diff
@@ -29,6 +29,8 @@ class OdsDatepickerController {
         this.component.datepickerInstanceAccessor?.setDate(newValue);
         this.component.emitDatepickerValueChange(newValue, oldValue ? oldValue : null);
       }
+      this.component.el.focus();
```

Comment:
can you add a comment with the explanation why both call are necessary

Reply:
I don't know why